R/vars_manager.R

```diff
@@ -147,6 +165,84 @@ get_vector_var <- function(
     ))
 }
 
+get_list_var <- function(
+    obj,
+    name,
+    no_preview=FALSE,
+    page_size=DEFAULT_PAGE_SIZE,
+    page=0,
+    filters=NULL
+) {
```

Comment:
I separated the `list` variable parsing from the `vector` variables, since the lists can be `named lists` and can have many different types of data stored within them (and that causes issues with sorting)